src/parser/bison_parser.y

```diff
@@ -374,24 +383,65 @@ execute_statement:
 
 /******************************
  * Import Statement
+ * IMPORT FROM TBL FILE 'test/students.tbl' INTO students
+ * COPY students FROM 'test/students.tbl' (WITH FORMAT TBL)
  ******************************/
 import_statement:
 		IMPORT FROM import_file_type FILE file_path INTO table_name {
-			$$ = new ImportStatement((ImportType) $3);
+			$$ = new ImportStatement($3);
 			$$->filePath = $5;
 			$$->schema = $7.schema;
 			$$->tableName = $7.name;
 		}
+	|	COPY table_name FROM file_path opt_file_type {
+			$$ = new ImportStatement($5);
+			$$->filePath = $4;
+			$$->schema = $2.schema;
+			$$->tableName = $2.name;
+		}
 	;
 
 import_file_type:
-		CSV { $$ = kImportCSV; }
+		IDENTIFIER {
+			if (strcasecmp($1, "csv") == 0) {
+			 	$$ = kImportCSV;
+			} else if (strcasecmp($1, "tbl") == 0) {
+			 	$$ = kImportTbl;
+			} else if (strcasecmp($1, "binary") == 0) {
```

Comment:
Is 'binary' a postgres thing? Do we want to support 'bin' as well, given that that is the file extension?

Reply:
Exactly, Postgres knows text, csv and binary as formats. I think bin is also a quite handy option, so I'm gonna add it